tests/ui/js/storyTelling-page-storyEditTests.js

```diff
@@ -14,7 +14,12 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     fluid.defaults("sjrk.storyTelling.page.testStoryEdit", {
         gradeNames: ["sjrk.storyTelling.page.storyEdit"],
         pageSetup: {
-            resourcePrefix: "../.."
+            resourcePrefix: "../..",
+            savingEnabled: false
```

Comment:
Again, this only tests the case when savingEnabled === false. The "true" case also needs to be tested. The same comment applies to other UI tests. Thanks.